docs/Community_IRC.md

```diff
@@ -30,13 +30,19 @@ In order to enter main `#armbian` channels registration with Nickserv is mandato
 Simply click here: [https://discord.com/invite/armbian](https://discord.com/invite/armbian)  
 
 Channels starting with `#armbian-` are relayed between Discord and Libera IRC so it does not matter if you join IRC or Discord as both ends receive your messages. Check `#welcome-and-rules` for more information.
-The main `#armbian` channel and `#armbian-announcements` are relayed between Discord, Libera and OFTC.
+The main `#armbian` channel and `#armbian-announcements` are relayed between Discord, Libera, OFTC and Matrix.
+
+### Matrix (closed beta)
+
+- Server: `matrix.armbian.com`
```

Comment:
I would propose autoredirection on web to www.armbian.com or this documentation.

Reply:
added